lib/db_utils.go

```diff
@@ -743,7 +732,7 @@ func StatePrefixToDeSoEncoder(prefix []byte) (_isEncoder bool, _encoder DeSoEnco
 		return true, &ValidatorEntry{}
 	} else if bytes.Equal(prefix, Prefixes.PrefixValidatorByStake) {
 		// prefix_id:"[79]"
-		return true, &PKID{}
+		return false, nil
```

Comment:
this is an important fix, good catch.

Reply:
Yeah. The value was a PKID until we now only store the ValidatorPKID in the key for this index.